### ISSUE TITLE
Question regarding annotation

### ISSUE THREAD

**Issue by hkaspersen** [oschwengers/bakta#102]
Hello!
I have annotated a few of my complete genomes with Bakta version 1.3.3 and bakta db version 3.1, using the following command:

```
bakta --threads 4 --prefix $outfile --complete --output ${outfile} --db $db $input
```
I noticed one of the annotations were "lysozyme", and I wanted to make sure this was correct, so I checked the accession number for the annotation (WP_032448305.1), and blasted the sequence itself to double-check. The weird thing is that the RefSeq accession number gives me MULTISPECIES: MliC family protein, which was also reflected in the BLAST search. I was wondering why the annotation itself said "lysozyme", when it clearly is a lysozyme-resistance gene instead? This is very confusing as they are essentially two opposite genes, functionally at least.

These are the reference IDs in the gff file:
RefSeq:WP_032448305.1
SO:0001217
UniParc:UPI000460FFF9
UniRef:UniRef100_A0A7U2DM59
UniRef:UniRef50_A0A1Q6AMZ2
UniRef:UniRef90_A0A7U2DM59

Thanks in advance!

**Comment by oschwengers** [comment]
Hi Håkon,
thanks for reaching out and reporting. This is indeed a wrong functional annotation. Unfortunately, this is due to a false annotation in the related UniProt UniRef100 und UniRef90 clusters:
- https://www.uniprot.org/uniref/UniRef100_A0A7U2DM59
- https://www.uniprot.org/uniref/UniRef90_A0A7U2DM59

In the [UniRef90] cluster, you can see that the seed sequence [A0A7U2DM59](https://www.uniprot.org/uniprot/A0A7U2DM59) is annotated differently ("Lysozyme") from all other members of this cluster, which are annotated as "MliC family protein" just like the RefSeq [WP_032448305.1](https://www.ncbi.nlm.nih.gov/protein/WP_032448305.1/) entry.

This is not the only instance where the UniRef90 seed sequence has a wrong or non-optimal annotation and unfortunately, there is not much we can do about it on a short term. However, this is something I'd like to address in the midterm by incorporating further annotation resources like KEGG, eggNOG and others.

Since this relates to the pre-annotation of the database and is not an actual bug in terms of the source code, I'll change the `bug` by an `enhancement` label but will keep this open until an enhancement of the pre-annotation workflow is in place.

Thanks and best regards!

**Comment by oschwengers** [comment]
Hi @hkaspersen ,
FYI, I just released version [v1.5.0](https://github.com/oschwengers/bakta/releases/tag/v1.5.0) introducing an improved pre-annotation of the PSC based on NCBIfams. A first quick test showed that in your case (above) the Bakta annotation is now in line with NCBI's annotation.

Hence, I'll close this for now but, please, do not hesitate to re-open this or another issue in case you see further room for improvements.

Again, thanks a lot!
